### PR TITLE
Feature/issue#128

### PR DESCRIPTION
**Pull request by oraclerob**
### Description
Puts a "copy project" icon on the Frontend edit screen.

### Benefits
Enables upstream users to copy projects so backend dashboard access is not required.

### Possible drawbacks
Possible security - however the ajax call that calls it is already public.

### CODE REVIEW

File: includes/class-up-frontend-output.php [hunk 1]
```diff
@@ -530,6 +544,17 @@ public function enqueues()
...
+        
+        //wp_enqueue_script('upstream-frontend-copy-project', UP_COPY_PROJECT_URL . '/assets/js/upstream-copy-project.js',
+        //    ['jquery'], UP_COPY_PROJECT_VERSION);
+
+        wp_enqueue_script('upstream-frontend-copy-project', UP_FRONTEND_PLUGIN_URL . '/assets/js/frontend-copy-project.js',
+            ['jquery'], UP_FRONTEND_VERSION);
+
+        wp_localize_script('upstream-frontend-copy-project', 'upstreamCopyProjectLangStrings', [
+            'ERR_INVALID_RESPONSE' => __('Invalid response', 'upstream-frontend-copy-project'),
+            'ERR_UNABLE_TO_COPY'   => __('It wasn\'t possible to copy this project.', 'upstream-frontend-copy-project'),
```
Comment (andergmartins) [inline includes/class-up-frontend-output.php]: @oraclerob just a minor detail. "project" is a term which can be customised by admin, so we can't use it here. You need to use the `upstream_project_label()` there.

Comment (oraclerob) [reply includes/class-up-frontend-output.php]: @andergmartins I've pushed the change up, also added a "confirm" button for copy projects.

**Comment by stevejburge** [comment]
Thanks so much @oraclerob 

UpStream 1.24.4 is already packaged and in testing, but we'll make sure this makes our next release too

File: includes/class-up-frontend-output.php (hunk 1 above)
Comment (andergmartins) [reply includes/class-up-frontend-output.php]: Thanks @oraclerob

**Pull request merged**